src/Provider/GoogleMaps/Model/GoogleAddress.php

```diff
@@ -116,7 +116,7 @@
      *
      * @return GoogleAddress
      */
-    public function withId(string $id = null): self
```

Comment:
Technically this is not really accurate anways. It should be `GoogleAddress` instead since it's not the same instance that is returned when calling these methods.

Reply:
Writing `:GoogleAddress` or `:self` is the same. `:self` means the same class. Not the same object.

Reply:
OK, you are right. So far I had in mind that this was the native `@return $this` solution.